### DISCUSSION TITLE
Use a cronjob to run sync command using a docker container

### DISCUSSION THREAD

**Discussion by kbgvirus** [Taxel/PlexTraktSync#906] [Support] [answered]
Hi,

Not sure if this is the correct place to document this, if not please advise where I can post this so it can help someone with the below.

The "watch" command does not update your collection.

I have PlexTraktSync running in a Linux Docker container and have set up the below cronjob to execute the sync command:

```
sudo crontab -e 
*/30 * * * * sudo docker exec trakt_plextraktsync_1 plextraktsync sync --sync all >/dev/null 2>&1
```
You can adjust the times to your liking.
**Please Note** my docker instance is called _trakt_plextraktsync_1_ please change to your instance name

You can use https://crontab-generator.org/ to help with generating the correct cronjob

**Comment by stevietv** [comment, marked as answer]
The Readme watch section (https://github.com/Taxel/PlexTraktSync#watch) already describes that the watch command does not update your collection.

for your crontab example, you could do a PR to add this to the README here https://github.com/Taxel/PlexTraktSync#setup

**Reply by kbgvirus** [reply]
Done, Thank you for pointing me in the right direction. If I see something missing that isn't in the read me that can help others will be sure to add it.

#907

**Comment by simonc56** [comment]
An other way is to use a docker scheduler like [Ofelia](https://github.com/mcuadros/ofelia) or [Chadburn](https://github.com/PremoWeb/chadburn) instead of cron.
I think I will add this idea in readme file -> https://github.com/Taxel/PlexTraktSync/pull/928/
